package.json

```diff
@@ -40,7 +40,8 @@
     "sinon": "^9.0.0",
     "stringify": "^5.1.0",
     "typescript": "^4.1.5",
-    "watchify": "^3.7.0"
+    "watchify": "^3.7.0",
+    "yalc": "^1.0.0-pre.50"
```

Comment:
This version number looks like we accidentally added a beta, but I confirmed that it is the current stable version.